libs/langgraph/langgraph/pregel/__init__.py

```diff
@@ -1825,6 +1893,14 @@ def _prepare_next_tasks(
             continue
         if for_execution:
             if node := processes[packet.node].get_node():
+                triggers = [TASKS]
+                metadata = {
+                    "langgraph_step": step,
+                    "langgraph_node": packet.node,
+                    "langgraph_triggers": triggers,
+                    "langgraph_task_idx": len(tasks),
+                }
+                task_id = str(uuid5(UUID(checkpoint["id"]), json.dumps(metadata)))
```

Comment:
Is it guaranteed that metadata is json serializable?  I think it is but wanted to check

Reply:
I created it just above, right? it's just strings and numbers

Reply:
ooops